test/src/unit-cppapi-enumerations.cc

```diff
@@ -58,6 +58,22 @@ struct CPPEnumerationFx {
 const static std::string enmr_name = "an_enumeration";
 const static std::string dump_name = "enumeration_dump_test.txt";
 
+TEST_CASE_METHOD(
```

Comment:
Do we expect code changes from an "update history" PR?

Reply:
Separate commit, it’s the backport of 4362